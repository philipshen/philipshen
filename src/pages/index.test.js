import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import IndexPage from './index.js'

const renderer = new ShallowRenderer()

describe('Index', () => {

  it('Matches snapshot', () => {
    const tree = renderer.render(<IndexPage />)
    expect(tree).toMatchSnapshot()
  })

})