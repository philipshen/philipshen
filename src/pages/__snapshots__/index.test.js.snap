// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Matches snapshot 1`] = `
<Layout
  currentHeaderSection={0}
  shouldHideHeader={false}
>
  <Home />
  <ForwardRef
    id="career"
  >
    <Waypoint
      bottomOffset="0px"
      fireOnRapidScroll={true}
      horizontal={false}
      onEnter={[Function]}
      onLeave={[Function]}
      onPositionChange={[Function]}
      topOffset="0px"
    />
  </ForwardRef>
  <Career />
  <ForwardRef
    id="portfolio"
  >
    <Waypoint
      bottomOffset="0px"
      fireOnRapidScroll={true}
      horizontal={false}
      onEnter={[Function]}
      onLeave={[Function]}
      onPositionChange={[Function]}
      topOffset="0px"
    />
  </ForwardRef>
  <Portfolio />
  <ForwardRef
    id="blog"
  >
    <Waypoint
      bottomOffset="0px"
      fireOnRapidScroll={true}
      horizontal={false}
      onEnter={[Function]}
      onLeave={[Function]}
      onPositionChange={[Function]}
      topOffset="0px"
    />
  </ForwardRef>
  <Blog />
</Layout>
`;
